### PR TITLE
[FIX] base_ubl: PartyName fix migration to 12.0

### PR DESCRIPTION
**Pull request by SimoneVagile**
See: https://github.com/OCA/edi/pull/166

### CODE REVIEW

File: sale_order_import_ubl/tests/files/UBL-Order-2.0-Example_no-party-name.xml [hunk 1]
```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SimoRubi) [inline sale_order_import_ubl/tests/files/UBL-Order-2.0-Example_no-party-name.xml:1]: The module `sale_order_import_ubl` does not exist in `12.0` yet, here you are just adding this file and the one below, this is not correct.
You should wait for https://github.com/OCA/edi/pull/102 to be merged before proposing this change

**Review by SimoRubi — CHANGES_REQUESTED** [review]
Thanks for your contribution!
Unfortunately, this has to wait until https://github.com/OCA/edi/pull/102 is merged

**Comment by SimoneVagile** [comment]
Depends on:

#102

**Comment by SimoneVagile** [comment]
Depends on:

- [ ] #102

File: sale_order_import_ubl/tests/files/UBL-Order-2.0-Example_no-party-name.xml (hunk 1 above)
Comment (SimoneVagile) [reply sale_order_import_ubl/tests/files/UBL-Order-2.0-Example_no-party-name.xml:1]: https://github.com/OCA/edi/pull/206#issuecomment-638769051

**Pull request closed without merging**